js/epics/cantieri.js

```
@@ -302,33 +302,31 @@ module.exports = {
-                            if (response.data.features.length === 0) {
-                                return Rx.Observable.of([]);
+                                }));
                             }
-                            return Rx.Observable.of(error({
-                                uid: ERROR_LOAD_CANTIERI_AREAS,
-                                title: "warning",
-                                message: "cantieriGrid.notification.errorLoadCantieriAreas",
-                                action: {
-                                    label: "cantieriGrid.notification.confirm"
-                                },
-                                autoDismiss: 3,
-                                position: "tr"
-                            }));
+                            return Rx.Observable.of([]);
                         }),
                     // fetch checked elements features
-                    Rx.Observable.fromPromise(axios.post(store.getState().cantieri.geoserverUrl + '?service=WFS&outputFormat=json&request=getFeature', getElementsFilter(store.getState().cantieri.checkedElements, store.getState().cantieri.elementsLayerName), {
-                                    timeout: 60000,
-                                    headers: {'Accept': 'application/json', 'Content-Type': 'application/json'}
-                                }).then(r => r.data.features))
-                ).concatMap(([areas, elements]) => {
+                    Rx.Observable.fromPromise(axios.post(store.getState().cantieri.geoserverUrl + '?service=WFS&outputFormat=json&request=getFeature',
+                        getElementsFilter(store.getState().cantieri.checkedElements, store.getState().cantieri.elementsLayerName), {
+                            timeout: 60000,
+                            headers: {'Accept': 'application/json', 'Content-Type': 'application/json'}
+                        }).then(r => r.data.features)).catch( () => Rx.Observable.of([]))
```

don't eat the exception